.github/workflows/CoverageCollection.yml

```diff
@@ -35,10 +35,15 @@ on:
         required: false
         default: '-r tests/requirements.txt'
         type: string
+      test_directory:
+        description: 'Path to the directory containing tests (test working directory).'
```

Comment:
The working directory does not need to be the one containing the tests. Therefore, I would use `workdir` for this input variable.